selfdrive/car/nissan/values.py

```diff
@@ -6,7 +6,8 @@
 
 class CAR:
   XTRAIL = "NISSAN X-TRAIL 2017"
-  LEAF = "NISSAN LEAF 2018"
+  LEAF_GB = "NISSAN LEAF 2018" # Glovebox
+  LEAF_IC = "NISSAN LEAF 2018 Instrument Cluster"
```

Comment:
put the explanatory comment above the LEAF_IC line

Reply:
Done